core/src/main/java/com/linecorp/armeria/common/DefaultRpcResponse.java

```diff
@@ -84,27 +82,4 @@ public boolean cancel(boolean mayInterruptIfRunning) {
         final boolean updated = !isDone() && completeExceptionally(new CancellationException());
         return updated || isCancelled();
     }
-
-    @Override
-    public String toString() {
```

Comment:
didn't see any reason to override this. Please let me know if there's one. I will put this back.

Reply:
I think it's because `CompletableFuture.toString()` ..

- Adds object hash code, which is somewhat verbose.
- Does not add the value.

Reply:
Thanks for the explanation. Let me add it back. 😄